packages/client/src/objectSet/ObjectSet.ts

```diff
@@ -97,11 +101,22 @@ export interface BaseObjectSet<Q extends ObjectOrInterfaceDefinition> {
     ...objectSets: ReadonlyArray<ObjectSet<Q>>
   ) => ObjectSet<Q>;
 
-  pivotTo: <L extends LinkNames<Q>>(type: L) => ObjectSet<LinkedType<Q, L>>;
+  pivotTo: <L extends LinkNames<Q>>(type: L) => BaseObjectSet<LinkedType<Q, L>>;
 
   subscribe: (listener: ObjectSetListener<Q>) => () => void;
 }
 
+export interface BaseObjectSet<
```

Comment:
Ah so you made a separate BaseObjectSet type so that we only allowed get calls when we came off like the base or a pivot, and not wheres or anything?

Reply:
Correct. This is the same approach that legacy-client took. Next PR will give it to us at `object.$link.multiLinkType` as well.